index.d.ts

```diff
@@ -0,0 +1,5 @@
+declare module 'fluture-sanctuary-types' {
+  export const env: ReadonlyArray<any>;
+  export const FutureType: any
+  export const ConcurrentFutureType: any
+}
```

Comment:
Could you add a newline at the end of the file?

Reply:
After modifying the file you can run `git add index.d.ts && git commit --amend`. :)

Reply:
just out of curiosity, why is the new line required 😮 ?

Reply:
Primarily for consistency with other files, but secondarily so that when you print the file on the terminal it looks nice, and so GitHub doesn't show the ugly icon at the end of the file.

Reply:
[What's the point in adding a new line to the end of a file?][1]


[1]: https://unix.stackexchange.com/q/18743

Reply:
I did not know that @davidchambers :)

Reply:
thank you 🙏